nerfstudio/configs/external_methods.py

```diff
@@ -152,6 +152,29 @@ class ExternalMethod:
     )
 )
 
+# K-Planes
```

Comment:
currently it says K-Panes here and in the instruction below! otherwise lgtm :)

Reply:
whoops :)

Reply:
fixed now!